thunder/executors/torch_compile.py

```diff
@@ -75,23 +80,25 @@ def torch_interpreted_func(*args):
     # TODO: issue "Try using _transform_for_operator_executor_execution for
     # torch.compile executor"
     torch_trace = trace(inline_trace=False)(torch_interpreted_func, *sorted_unique_inputs)
-    compiled_func = torch.compile(torch_trace.python_callable())
+    trace_callable = torch_trace.python_callable(include_decorators=False)
+    compiled_func = torch.compile(trace_callable, fullgraph=True)
```

Comment:
What's the benefit of `fullgraph=True`? Should we use some other flags like reduce-overhead?

Reply:
My end goal is getting #140 to work. I was facing issues with that + LitGPT and my first reaction was to try to get fullgraph compilation to work.

Since torch.compile has good operator coverage, it should be safe to set this today. That should also enable more opportunities for optimization on their side and surface decisions in our implementation that disable it.

In the future I would suggest that we expose fullgraph and the other backend flags (like reduce-overhead) to the thunder user

Reply:
Opened #281